modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -98,9 +98,9 @@ def new_notice_of_disagreement
 
   def api_version
```

Comment:
this is the one spot that I was split on what to do, since this is clearly deriving the version value from the api version, but then is passing it into the the PDF generator that, if I'm understanding it correctly, is going off of the PDF version ([ref](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb#L31))

Reply:
I think it's fine to leave a-is. The way API & PDF versions are pretty closely tied together means there likely wasn't going to be a 100% clean path from API version to PDF version in this sort of case.

Reply:
my small workaround: [add a comment and quick variable ](https://github.com/department-of-veterans-affairs/vets-api/pull/10385/commits/fd3f2f8f109c090301404453e0baa59a58b21ac4)to make that connection less confusing at a glance